spec/javascripts/govuk_publishing_components/analytics/google-analytics-universal-tracker.spec.js

```diff
@@ -350,4 +354,8 @@ describe('GOVUK.GoogleAnalyticsUniversalTracker', function () {
   function pageWantsPostcodesStripped () {
     $('head').append('<meta name="govuk:static-analytics:strip-postcodes" value="does not matter" />')
   }
+
+  function pageWantsVaccinationStatusStripped () {
+    $('head').append('<meta name="govuk:static-analytics:strip-vaccination-status" value="does not matter" />')
```

Comment:
I get from the test that one possible status for this is `fully-vaccinated` - just out of curiousity, what are the other possible statuses?